daml-foundations/daml-ghc/daml-prim-src/DA/Types.daml

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+
+daml 1.2
+
+-- | A module containing all the standard types from the base libraries,
+--   so they have nice names when used from Java or similar.
+module DA.Types where
```

Comment:
```suggestion
module DA.Types
  ( Either (..)
  ) where
```
The `TupleN` types are not supposed to be used in DAML anyway and DAML-LF does not care about export statements.